awx/main/management/commands/create_preload_data.py

```diff
@@ -42,6 +42,16 @@ def handle(self, *args, **kwargs):
                                               },
                                               created_by=superuser)
                 c.admin_role.members.add(superuser)
+                public_galaxy_credential = Credential(
```

Comment:
I'm on my cell phone. I assume this only runs on fresh installs and the migration will create this otherwise.

Reply:
Yep, here:

https://github.com/ansible/awx/pull/7817/files#diff-990db3abe66cd302b90fe644303d7879R50